contracts/erc20-gateway/ERC20Gateway.sol

```diff
@@ -26,6 +27,21 @@ import "../proxies/MasterCopyNonUpgradable.sol";
  */
 contract ERC20Gateway is MasterCopyNonUpgradable, MessageBus, ERC20GatewayBase {
 
+    /* Events */
+
+    /* Emitted when deposit intent hash is declared. */
+    event DepositIntentDeclared(
+        bytes32 messageHash,
+        address valueToken,
+        uint256 amount,
+        uint256 nonce,
+        address beneficiary,
+        uint256 feeGasPrice,
+        uint256 feeGasLimit,
+        address depositor
```

Comment:
`depositor` can come after `valueToken` in ordering.

Reply:
Done.